src/applications/mhv-medical-records/components/shared/NewRecordsIndicator.jsx

```diff
@@ -162,9 +162,36 @@ const NewRecordsIndicator = ({
   };
 
   return (
-    <div className="vads-u-margin-y--2  no-print" id="new-records-indicator">
-      {content()}
-    </div>
+    <>
+      <div className="vads-u-margin-y--2 no-print" id="new-records-indicator">
+        {content()}
+      </div>
+      <div className="print-only">
+        {!lastSuccessfulUpdate && (
+          <span>
+            <p>
+              <b>
+                Your records may not be up to date.
+                <br />
```

Comment:
Rather than use a `<br />` tag here, can these two lines be two separate paragraphs (`<p>`)?